SECURITY.md

```diff
@@ -0,0 +1,22 @@
+# Security Policy
+
+## Supported Versions
+
+| DOMjudge Version | Supported          | PHP version supported |
+| ---------------- | ------------------ | --------------------- |
+| 8.x.x            | :warning:          | 7.4-8.1               |
+| 8.0.x            | :white_check_mark: | 7.2-8.0               |
+| 7.3.x            | :white_check_mark: | 7.2-7.4               |
+| < 7.3            | :x:                | :x:                   |
+
+## Reporting a Vulnerability
```

Comment:
Move this up?

Reply:
Its the default template, so I rather keep it default.

Reply:
I didn't know.